src/connection.js

```diff
@@ -348,6 +352,102 @@ class Connection {
 
 		this.packets.push(packet);
 	}
+
+	/**
+	 *
+	 * @param {Buffer} data Raw RMC payload data from HokakuCTR
+	 */
+	handleRawRMC(data) {
+		// TODO - Refactor handlePacket to also use this function? To reduce duplicate code?
+		const stream = new Stream(data, this);
+		const revision = stream.readUInt8();
+
+		if (revision !== 1) {
+			throw new Error(`Found packet with unsupported HokakuCTR version. Expected 1, got ${revision}`);
+		}
+
+		const titleID = stream.readUInt64LE().toString(16).padStart(16, '0').toUpperCase();
+		const flags = stream.readUInt8();
+		const message = new RMCMessage(stream.readRest());
+
+		const isResponseRMCMessage = (flags & 0b00000001) !== 0;
+
+		if (!this.title.name) {
+			for (const title of titles) {
+				if (title.title_ids?.includes(titleID)) {
+					this.title = title;
```

Comment:
We are using `this.accessKey` when checking for game-specific protocols

```suggestion
					this.title = title;
					this.accessKey = title.access_key;
```